src/Helper/HeadLink.php

```diff
@@ -387,15 +387,15 @@ public function createDataStylesheet(array $args)
             return false;
         }
 
-        if (0 < count($args)) {
+        if (! empty($args)) {
```

Comment:
All these `! empty($args)` can be simplified to `$args` in the context of boolean expressions